### PR TITLE
Feature/doc

### PR DESCRIPTION
**Pull request by pcrespov**
Started a collection of re-editable diagrams with examples and descriptions of the service architecture.

### CODE REVIEW

File: services/web/client/source/class/qxapp/Preferences.js [hunk 1]
```diff
@@ -77,9 +77,13 @@ qx.Class.define("qxapp.Preferences", {
...
 
-      // const url = qxapp.utils.Placeholders.getIcon("fa-user", 200);
       let img = new qx.ui.basic.Image().set({
-        source: qxapp.utils.Placeholders.getGravatar(email.getValue() || "bizzi@simcore.io", 200)
+        source: qxapp.utils.Placeholders.getGravatar(
+            email.getValue() ||
+                ('bizzy' +
+                 '@' +
+                 'itis.ethz.ch'),
+            200)
```
Comment (odeimaiz) [inline services/web/client/source/class/qxapp/Preferences.js]: We use double quotes

File: services/web/client/source/class/qxapp/desktop/NavigationBar.js [hunk 2]
```diff
@@ -36,14 +36,17 @@ qx.Class.define("qxapp.desktop.NavigationBar", {
       flex: 1
     });
 
-    let userLbl = new qx.ui.basic.Label("@bizzi").set({
+    const dummy_user="bizzy";
+    let userLbl = new qx.ui.basic.Label("@" + dummy_user).set({
```
Comment (odeimaiz) [inline services/web/client/source/class/qxapp/desktop/NavigationBar.js]: camelCase

File: services/web/client/source/class/qxapp/desktop/NavigationBar.js [hunk 3]
```diff
@@ -36,14 +36,17 @@ qx.Class.define("qxapp.desktop.NavigationBar", {
...
       minWidth: 20
     });
     this.add(userLbl);
 
     let userBtn = this.__createUserBtn();
     userBtn.set(commonBtnSettings);
-    userBtn.setIcon(qxapp.utils.Placeholders.getGravatar("bizzi@simcore.io", 32));
+    userBtn.setIcon(qxapp.utils.Placeholders.getGravatar(
+        dummy_user + '@' + 'itis.ethz.ch',
+        32));
```
Comment (odeimaiz) [inline services/web/client/source/class/qxapp/desktop/NavigationBar.js]: Double quotes + camelCase

**Comment by ehzastrow** [comment]
looks good to me.

File: services/web/client/source/class/qxapp/desktop/NavigationBar.js (hunk 3 above)
Comment (pcrespov) [reply services/web/client/source/class/qxapp/desktop/NavigationBar.js]: 👍 Odei! I have been "contaminated" by python ...

Comment (odeimaiz) [reply services/web/client/source/class/qxapp/desktop/NavigationBar.js]: Bitten by python...

**Pull request merged**